docs/concepts/pdp/overview.mdx

```diff
@@ -0,0 +1,248 @@
+Read about [the various layouts you can deploy the PDP with here](/how-to/deploy/overview#pdp-deployments).
+
+There are several ways you can deploy the PDP:
+
+
+### Managed Cloud PDP
+
+Permit.io provides a managed Cloud PDP at `https://cloudpdp.api.permit.io` which is available globally.
+
+When first trying out Permit, you can connect quickly with the cloud PDP.
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<Tabs groupId="language">
+<TabItem value="python" label="Python">
+
+```python
+from permit import Permit
+
+permit = Permit(
+    token="[your-api-key]",
+    pdp="https://cloudpdp.api.permit.io",
+)
+```
+
+</TabItem>
+<TabItem value="nodejs" label="Node.js">
+
+```typescript
+import { Permit } from "permit";
+
+const permit = new Permit({
+  token: "[your-api-key]",
+  pdp: "https://cloudpdp.api.permit.io",
+});
+```
+</TabItem>
+<TabItem value="go" label="Go">
+
+```go
+package main
+
+import "github.com/permitio/permit-golang/pkg/permit"
+import "github.com/permitio/permit-golang/pkg/config"
+
+func main() {
+    PermitConfig := config.NewConfigBuilder("[your-api-key]")
+        .WithPdpUrl("https://cloudpdp.api.permit.io")
+        .Build()
+    Permit := permit.New(PermitConfig)
+}
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+import io.permit.sdk.Permit;
+import io.permit.sdk.PermitConfig;
+
+Permit permit = new Permit(
+    new PermitConfig.Builder("[your-api-key]")
+        .withPdpAddress("https://cloudpdp.api.permit.io")
+        .build()
+);
+```
+
+</TabItem>
+<TabItem value="dotnet" label="DotNet">
+
+```csharp
+using Permit;
+
+Permit permit = new Permit(
+  "[your-api-key]",
+  "https://cloudpdp.api.permit.io"
+);
+```
+
+</TabItem>
+<TabItem value="ruby" label="Ruby">
+
+```ruby
+require 'permit'
+
+permit = Permit.new(
+  "[your-api-key]",
+  "https://cloudpdp.api.permit.io"
+)
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+For production deployments, we recommend deploying a local PDP to minimize network latency and ensure high availability.
+:::
+
+:::info
+Custom cloud PDP deployments are available to enterprise tier customers.
+Those can include different regions, cloud providers, custom SSL/TLS configurations or other specifications.
+
+Please reach out to us at [support@permit.io](mailto:support@permit.io), or schedule a call via this link: [https://calendly.com/permitio/](https://calendly.com/permitio/)
+:::
+
+
+### Hosted as a Sidecar
+
+The simplest way to integrate PDPs into a microservices architecture is as sidecars;
+meaning each microservice has a sidecar container next to it which it can query for policy.
+
+For more information, see the [sidecar deployment guide](/how-to/deploy/overview#sidecar).
+
+```mermaid
```

Comment:
in the graphs it looks like the pdp is interacting with the cloud, I think it is better to change the arrow to be from the cloud to the PDP to show that the pdp doesn't query the cloud but the cloud updates it

Reply:
I tried doing it, but it messed up the chart. I've now found bidirectional arrows and I think it fits best